Gemfile

```diff
@@ -25,6 +25,14 @@ group(:development, :test) do
   gem 'cheffish', "~> 1.3", "!= 1.3.1"
 
   gem 'ruby-shadow', :platforms => :ruby unless RUBY_PLATFORM.downcase.match(/(aix|cygwin)/)
+
+  # For external tests
+  gem 'chefspec', github: 'jkeiser/chefspec', branch: 'jk/chefspec-12.5'
+  gem 'chef-sugar'
+  gem 'poise', github: 'poise/poise', branch: 'deeecb890a6a0bc2037dfb09ce0fd0a8931519aa'
```

Comment:
Hmm, those gems get added via [the Halite Gemfile](https://github.com/poise/halite/blob/master/Gemfile#L31-L34), but I don't see anything in your Gemfile munger that should be disrupting that.